### ISSUE TITLE
[QUESTION] Is it reasonable to use a restart file that is generated from a tropchem spin-up to drive a RRTMG?

### ISSUE THREAD

**Issue by FeiYao-Edinburgh** [geoschem/geos-chem#487]
Hi there,

I have found that when creating a RRTMG rundir, `gcCopyRunDirs` links the restart file to that of tropchem (below). Also I note that tropchem and RRTMG both use tropchem chemistry. Hence I'd like to confirm is it  reasonable to use a restart file that is generated from a tropchem spin-up to drive a RRTMG? I have done a sample comparison and found that most variables contained in restart files generated from tropchem and RRTMG spin-ups are identical, although slight differences are also observed for `Chem_KPPHvalue` and `Chem_DryDepNitrogen`. If the answer is yes, then I can save model running time during spin-up, to some extent though. 
```
          } elsif ( $sim =~ m/RRTMG/ ) {
            $rstFile = "$rstRoot$rstPref$grid\_tropchem.nc";
```

**Comment by msulprizio** [comment]
Yes, we recommend spinning up your RRTMG simulation with a tropchem restart file. The RRTMG simulation in GEOS-Chem is a tropchem simulation that has been interfaced with RRTMG to save out additional diagnostics.

**Comment by FeiYao-Edinburgh** [comment]
Hum sorry I may not have clarified my demand.

Assume I want to run a RRTMG from 20160701 to 20200101. The first one and half years (20160701-2018010) are used as spin-up, and the last two years (20180101-20200101) are used for scientific analyses.

I have a tropchem restart file at 20130701 that I rename to 20160701. Then it is natural to use this renamed restart file to directly drive the RRTMG simulation from 20160701 to 2020010. Later on only 20180101 to 20200101 are taken for scientific analyses.

But I want to decrease the spin-up time by using a tropchem. That is, I use the 20160701 restart file to drive a tropchem for one and half years and hence collect a restart file at 20180101. Then I use this restart file at 20180101 to drive the RRTMG and collect datasets for scientific analyses from 20180101 to 20200101.

Long story short, for a RRTMG run, I want to split the run into: 1) spin-up but replaced by tropchem; 2) real run. Is it reasonable? Or I'd better to keep using RRTMG for spin-up?

**Comment by FeiYao-Edinburgh** [comment]
Never mind, I suppose tropchem and RRTMG should have similar running time, if I only collect Restart diagnostics during the spin-up. I believe I am all set and you can close this issue out. Thanks.

**Comment by msulprizio** [comment]
Yes, that will work.
